ci-operator/config/openshift/cluster-dns-operator/openshift-cluster-dns-operator-master.yaml

```diff
@@ -75,6 +75,13 @@ tests:
   steps:
     cluster_profile: aws
     workflow: openshift-e2e-aws-single-node
+- as: e2e-aws-ovn-techpreview
+  optional: true
```

Comment:
Made the CI test optional for merging as there are higher chances of unrelated failures due to all tech-preview features getting enabled.